apis/nucleus/src/components/Cell.jsx

```diff
@@ -571,7 +581,7 @@ const Cell = forwardRef(
             position: 'relative',
             width: '100%',
             height: '100%',
-            ...(!disableCellPadding ? { padding: theme.spacing(1) } : {}),
+            ...(cellPadding ? { padding: cellPadding } : {}),
```

Comment:
Can we move this up to the <Paper>?  That one has a className defined in the API, making it easier to change with a targeted selector.

Reply:
I would suggest not moving it,
As this is the old padding (that will go away when we turn of the flag)